src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const items = document.querySelectorAll('li');
+const arr = [...items];
+
+const sortList = (employees) => {
+  const sorted = (item) => item.dataset['salary'].slice(1).replace(',', '.');
```

Comment:
```suggestion
  const getSalary = (item) => item.dataset['salary'].slice(1).replace(',', '.');
```
also, you have to compare them as numbers (do type conversion)